ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -48,7 +48,7 @@ library
                      , cardano-ledger-core
                      , cardano-prelude
                      , cardano-slotting
-                     , shelley-spec-ledger
+                     , cardano-ledger-shelley
```

Comment:
Would be nice if we can sort these (I think they were sorted before this change)